### PR TITLE
refactor: remove AdminExpr, make DDL expressions as normal GRPC requests

### PR DESCRIPTION
**Pull request by MichaelScofield**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

This PR is part of a series of changes to gradually adopt the Arrow Flight, and remove the old GRPC interface finally.

This PR removes AdminExpr (and the whole Admin service), make DDL expressions as normal GRPC requests.

## Checklist

- [ ]  I have written the necessary rustdoc comments.
- [ ]  I have added the necessary unit tests and integration tests.

## Refer to a related PR or issue link (optional)

#381

### CODE REVIEW

File: src/datanode/src/instance/flight.rs [hunk 1]
```diff
@@ -92,6 +93,7 @@ impl FlightService for Instance {
                 self.handle_query(query).await?
             }
             GrpcRequest::Insert(request) => self.handle_insert(request).await?,
+            GrpcRequest::Ddl(request) => self.handle_ddl(request).await?,
```
Comment (fengjiachun) [inline src/datanode/src/instance/flight.rs:96]: The recommended order is:
```
GrpcRequest::Insert
GrpcRequest::Query
GrpcRequest::Ddl
```
Keep the same order as defined in `database.proro`

**Review by fengjiachun — APPROVED** [review]

File: src/api/greptime/v1/ddl.proto [hunk 2]
```diff
@@ -5,17 +5,10 @@ package greptime.v1;
...
-
-message AdminExpr {
-  ExprHeader header = 1;
+// "Data Definition Language" requests, that create, modify or delete the database structures but not the data.
+// `DdlRequest` could carry more information than plain SQL, for example, the "table_id" in `CreateTableExpr`.
+// So create a new DDL expr if you need it.
+message DdlRequest {
   oneof expr {
     CreateTableExpr create_table = 2;
     AlterExpr alter = 3;
```
Comment (Fengys123) [inline src/api/greptime/v1/ddl.proto:14]:
```suggestion
    CreateTableExpr create_table = 1;
    AlterExpr alter = 2;
```

File: src/api/greptime/v1/greptime.proto [hunk 3]
```diff
@@ -12,11 +11,9 @@ service Greptime {
 
 message BatchRequest {
   RequestHeader header = 1;
-  repeated AdminRequest admins = 2;
   repeated DatabaseRequest databases = 3;
```
Comment (Fengys123) [inline src/api/greptime/v1/greptime.proto:14]:
```suggestion
  repeated DatabaseRequest databases = 2;
```

File: src/api/greptime/v1/greptime.proto [hunk 4]
```diff
@@ -12,11 +11,9 @@ service Greptime {
...
 }
 
 message BatchResponse {
-  repeated AdminResponse admins = 1;
   repeated DatabaseResponse databases = 2;
```
Comment (Fengys123) [inline src/api/greptime/v1/greptime.proto:18]:
```suggestion
  repeated DatabaseResponse databases = 1;
```

**Review by Fengys123 — APPROVED** [review]
LGTM

**Pull request merged**